src/routes/user.js

```diff
@@ -0,0 +1,22 @@
+import express from 'express';
+import getUsers, {
+  getUser, createUser, updateUser, deleteUser,
+} from './admin/user';
+
+const router = new express.Router();
+
+/**
+ * API for frontend to manage user updates.
+ */
+
+router.get('/:userId', getUser);
+
+router.get('/', getUsers);
```

Comment:
Is there a use case for this look up? It seems really expensive.

Reply:
Yes, this could get expensive for a lot of users. I see this as the start. We might want to add filtering parameters, e.g. region to narrow down the output.